backend/codesirius/codesirius/tests/test_email_or_username_backend.py

```
@@ -0,0 +1,62 @@
+        self.backend = EmailOrUsernameBackend()
+        self.user = User.objects.create_user(
+            username="testuser",
+            email="test@example.com",
+            password="testpass123",
+            first_name="Test",
+        )
+
+    def test_authenticate_with_username(self):
+        """Test authentication using username"""
+        user = self.backend.authenticate(
+            None, username="testuser", password="testpass123"
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)